packages/@aws-cdk/aws-sqs/lib/perms.ts

```diff
@@ -0,0 +1,7 @@
+export const QUEUE_GET_ACTIONS = [
```

Comment:
No need to expose these I think. Might as well put them in the `queue.ts` file and not `export` them.